src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/Modal/feedback-filter-modal.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, EventEmitter, Output, inject, signal } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { FormBuilder, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { RangeSliderComponent } from 'app/shared/range-slider/range-slider.component';
+import { FeedbackAnalysisService } from 'app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.service';
+import { ArtemisSharedCommonModule } from 'app/shared/shared-common.module';
+import { LocalStorageService } from 'ngx-webstorage';
+
+@Component({
+    selector: 'jhi-feedback-filter-modal',
+    templateUrl: './feedback-filter-modal.component.html',
+    imports: [RangeSliderComponent, ArtemisSharedCommonModule, ReactiveFormsModule],
+    providers: [FeedbackAnalysisService],
+    standalone: true,
+})
+export class FeedbackFilterModalComponent {
+    private localStorage = inject(LocalStorageService);
+    private activeModal = inject(NgbActiveModal);
+    private fb = inject(FormBuilder);
+    @Output() filterApplied = new EventEmitter<any>();
```

Comment:
We want to use signals from now on. Can you use them instead of the `@Output()`?